treeherder/webapp/api/performance_data.py

```diff
@@ -845,6 +852,16 @@ def list(self, request):
                     'new_stddev_pct': new_stddev_pct,
                     'base_retriggerable_job_ids': base_grouped_job_ids.get(base_sig_id, []),
                     'new_retriggerable_job_ids': new_grouped_job_ids.get(new_sig_id, []),
+                    # highlighted revisions is the base_revision and the other highlighted revisions is new_revision
+                    'graphs_link': self._create_graph_links(
+                        base_repository_name,
+                        new_repository_name,
+                        new_revision,
+                        base_revision,
+                        str(framework),
+                        str(interval),
```

Comment:
```suggestion
                        interval,
```

Reply:
for clarity when checking if value exists